app/core/database_service.py

```
@@ -0,0 +1,558 @@
+            "total_queries": 0,
+            "successful_queries": 0,
+            "failed_queries": 0,
+            "avg_execution_time": 0.0
+        }
+    
+    @asynccontextmanager
+    async def get_session(self) -> AsyncSession:
+        """Get database session with bulletproof error handling."""
+        session = None
+        try:
+            session = await anext(get_database())
```

Ruff (0.13.1)

Undefined name `anext`. Consider specifying `requires-python = ">= 3.10"` or `tool.ruff.target-version = "py310"` in your `pyproject.toml` file.

(F821)